### PR TITLE
Add gitter integration

### PR DESCRIPTION
**Pull request by rclanan**
At first, we setup slack, so we could move the internal chat so external users could join us. Slack doesn't allow guests to join in without invites (or a link to invite). Gitter.im appears to have a lot of the same integrations, open to guests, and a lot of popular open source projects are using it.

Here at Fusion, we have used it on a couple of other projects as well, and it seems to work pretty well.

### CODE REVIEW

File: .codecov.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+codecov:
```
Comment (BWortman) [inline .codecov.yml]: So we need all this just to integrate with gitter?

File: README.md [hunk 2]
```diff
@@ -106,3 +107,6 @@ This project is licensed under the MIT License - see the [LICENSE](LICENSE) file
 
 [travis-url]: https://travis-ci.org/fusionalliance/autorenter-angular1
 [travis-image]: https://travis-ci.org/fusionalliance/autorenter-angular1.svg?branch=development&style=flat-square
```
Comment (BWortman) [inline README.md]: Seems like we're adding a lot of "bling" to this...

Comment (rclanan) [inline README.md]: The "bling" isn't just bling.. it's basically adding hyperlinks for people to be able to get to say our gitter channel, or our travis build logs. Otherwsie, they have no way of knowing what we are integrating with to be able to find it.

File: .codecov.yml (hunk 1 above)
Comment (rclanan) [inline .codecov.yml]: So codecov, from what I can see didn't allow gitter integration through the web interface like they do for Slack. You have to add it to the configuration file. The "config" file that is being added here is the one that codecov spit out based on the settings that we have selected in the web interface. This allowed us to add the notifications to gitter. 

Short answer, yes.

**Comment by rclanan** [comment]
Apparently, there wasn't an issue on our side with CodeCov. CodeCov responded a bit ago and said that their queue was really backed up and almost 24 hours later it completed. I switched us over to coveralls.io instead. Worked very fast and easy to setup.

**Pull request merged**